app/src/main/java/co/nimblehq/compose/crypto/ui/preview/LoadingPreviewParameterProvider.kt

```diff
@@ -0,0 +1,11 @@
+package co.nimblehq.compose.crypto.ui.preview
+
+import androidx.compose.ui.tooling.preview.PreviewParameterProvider
+import co.nimblehq.compose.crypto.lib.IsLoading
+
+class LoadingPreviewParameterProvider : PreviewParameterProvider<IsLoading> {
+    override val values = sequenceOf(
+        false,
+        true
+    )
+}
```

Comment:
FYI: This will help us auto-generate states for loading

![image](https://user-images.githubusercontent.com/16315358/189025257-4a1e9ffe-deb9-4075-9353-cb82cac87818.png)